src/main.rs

```diff
@@ -36,13 +36,13 @@ fn main() -> PassResult<()> {
 
     let pm = PassManager::default_passes()?;
 
-    // list all the avaliable pass options when pass-help subcommand is used
-    if let Some(sub) = opts.sub {
+    // handle pass-help and format
+    if let Some(ref sub) = opts.sub {
         match sub {
             cmdline::Subcommand::Help(cmdline::Help { name }) => {
                 match name {
                     Some(n) => {
-                        if let Some(help) = pm.specific_help(&n) {
+                        if let Some(help) = pm.specific_help(n) {
```

Comment:
I think we should revert this change since the `ref` above seems unnecessary to me